src/reducers-actions/tokenRow.ts

```diff
@@ -1,6 +1,6 @@
 import { Actions } from 'reducers-actions'
 
-export const enum ActionTypes {
+export enum ActionTypes {
```

Comment:
storybook hates `const enum` though I think `enum` is safe as is here without `const`